ftw/referencewidget/converter.py

```diff
@@ -49,3 +50,27 @@ def toFieldValue(self, value):
     def toWidgetValue(self, value):
         if value:
             return '/'.join(value.getPhysicalPath())
+
+
+class ReferenceDataListWithChoiceConverter(converter.BaseDataConverter):
+    """Converter of IList of IRelationChoices."""
+
+    adapts(IList, IReferenceWidget)
+
+    def toFieldValue(self, value):
+        if not value:
+            return []
+        elif isinstance(value, unicode):
```

Comment:
I'm sorry I really don't know, I copied it from the code above. I guess if it's not a list of unicode it's a unicode. Probably the extracting from the request makes sure it is?? Since there are no other tests I'm not able to tell you. IMHO it's always a list, but who knows.